ArcadiaParties-SPA/src/app/user/user-info/user-info.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="flex-container">
+  <mat-card>
+    <mat-card-title style="text-align:center">Your information</mat-card-title>
+    <mat-tab-group mat-stretch-tabs>
+      <mat-tab label="General">
+        <mat-card-content>Identity: {{ (user$ | async).identity }} </mat-card-content>
```

Comment:
Only single `mat-card-content` should exist inside `mat-card`.

Reply:
I've fixed it